content/doc/book/system-administration/administering-jenkins-on-kubernetes.adoc

```diff
@@ -12,4 +12,176 @@ ifdef::backend-html5[]
 :toc: left
 endif::[]
 
-= Administering Jenkins on Kubernetes
\ No newline at end of file
+= Administering Jenkins on Kubernetes
+
+In this section, we will look at:
+
+* Upgrading from one version of Kubernetes to another.
+* How to upgrade the version of Kubernetes in your cluster.
+* Upgrade Kubernetes cluster maintained by the Jenkins Infrastructure project.
+* Upgrade Jenkins Inside a Docker Container.
+* Kubernetes Concepts.
+
+== Upgrading from one version of Kubernetes to another:
+
+Kubernetes (K8s) is an open-source system for automating deployment, scaling, and management of containerized applications.
+
+A Kubernetes cluster adds a new automation layer to Jenkins.
+Kubernetes makes sure that resources are used effectively and that your servers and underlying infrastructure are not overloaded.
+Kubernetes' ability to orchestrate container deployment ensures that Jenkins always has the right amount of resources available.
+
+=== How to upgrade the version of Kubernetes in your cluster:
+
+Upgrading your Kubernetes cluster is one of the tasks you must take on as an operator of Jenkins X. 
+Kubernetes comes out with link:https://kubernetes.io/releases[new releases] regularly, and will only maintain releases for the three latest versions.
+Therefore, if something breaks or there is a security issue on an older version, Kubernetes will not fix it, but ask you to upgrade.
+All the major cloud providers only support a given set of latest versions.
+
+Jenkins X currently supports Kubernetes versions 1.20 and 1.21.
+Version 1.22 introduces a lot of deprecations, and is not fully supported yet.
```

Comment:
I need help from the Kubernetes experts in the Jenkins community for this one.  I hesitate to state support for specific versions here because I believe it depends on the installation technique and the group of people maintaining that installation technique.  For example, I would expect that the helm chart maintainers would be the best ones to state the Kubernetes versions they test with the helm charts.  Likewise for those who maintain or adjust the YAML file definitions in the documentation.

@dduportal, @lemeurherve , and @smerle33 do you have ideas of better locations to declare the Kubernetes versions that are supported?
```suggestion
```

Reply:
I don't see a reason to list supported version if the scope is Jenkins itself: it works on any version.

However, for the helm chart, then the kube version is mandatory to check (see the recent bugs with kube 1.25 and our helm chart).
Same for the kubernetes plugin.

I whole-hearthdly agree with what you said @MarkEWaite : it depends on the installation technique (see my other message above)